### ISSUE TITLE
Cannot export URDF file

### ISSUE THREAD

**Issue by gy2256** [dfki-ric/phobos#266]
**Describe the bug**
When I click export button after all the setting nothing happened. 

**To Reproduce**
1. Click model
2. Hit "n"
3. Hit Export
4. Nothing happens. 

**Expected behavior**
URDF file will be produced. 

**Desktop (please complete the following information):**
 - OS: Mac OS Ventura (M1 Pro MacBook Pro)
 - Version 13.2
 - Blender: 3.4.0 (Apple Silicon Native)
 - Phobos 1.0.3

**Comment by hwiedPro** [comment]
Have you selected in the viewport, what you want to export?

**Comment by gy2256** [comment]
Yes. I pressed the 'a' and the model is selected. Nothing happened when I clicked 'a'.

**Comment by hwiedPro** [comment]
Is everything highlighted in the viewport after pressing 'a'?

**Comment by gy2256** [comment]
<img width="1015" alt="Screenshot 2023-02-01 at 3 46 04 PM" src="https://user-images.githubusercontent.com/4914320/216092178-17ec6e3e-ac3c-4f44-9390-549d48457762.png">
Thanks for the reply. This is what it looks like after I press 'a'.

**Comment by hwiedPro** [comment]
It seems you have neither created a link, nor set a modelname and version, nor parented this visual to that link. Therefore you haven't yet created any structure that could be exported to URDF.

BTW: I can't see in your screenshot whether you have defined the geometry but if it shall be a mesh, you are currently not exporting any meshes due to your export settings, therefore your URDF will be invalid in the end.

Please see https://github.com/dfki-ric/phobos/wiki/Modeling-Walkthrough on how to create robot models.

**Comment by hwiedPro** [comment]
I consider this as solved. Please feel free to reopen, if the issue persists.
